app/importers/fema_importer.rb

```diff
@@ -0,0 +1,44 @@
+require 'rubygems'
+require 'httparty'
+
+class FemaImporter
+  include HTTParty
+  base_uri 'https://gis.fema.gov/geoserver/FEMA'
+  format :json
+
+  QueryValues = {
+    'service' => 'WFS',
+    'version' => '1.0.0',
+    'request' =>' GetFeature',
+    'typeName' => 'FEMA:FEMANSSOpenShelters',
+    'maxFeatures' => 250,
+    'outputFormat' => 'json'
+  }.freeze
+
+
+  def self.shelters
+    fema_data = get('/ows', query: QueryValues)
+    fema_data['features'].map do |f|
+      props = f['properties']
+      data = {
+        longitude: f['geometry']['coordinates'][0],
+        latitude: f['geometry']['coordinates'][1],
+        source: "FEMA GeoServer, Shelter ID: #{props['SHELTER_ID']}, Org ID: #{props['ORG_ID']}, #{props['ORG_NAME']}",
+        shelter: props['SHELTER_NAME'],
+        address: "#{props['ADDRESS']}, #{props['CITY']}, #{props['STATE']} #{props['ZIP']}",
+        city: props['CITY'],
+        state: props['STATE'],
+        zip: props['ZIP'],
+        active: props['SHELTER_STATUS'] == 'OPEN' ? :yes : :no
```

Comment:
Is there still an open question between our archive and accepting? I know we did a bulk archive instead of a bulk not accepting and now wondering what is better. Might be a question for next run though. But wanted to clarify how we are approaching the 2 settings.

Reply:
Do we know what the enumeration of values for SHELTER_STATUS are from FEMA?  Is it a simple boolean [OPEN; CLOSED] or is there something more, like [OPEN; FULL; ???; CLOSED]?

I guess we could update the logic here to be more inverse:
```Ruby
active: props['SHELTER_STATUS'] == 'CLOSED' ? :no : :yes
accepting: props['TOTAL_POPULATION'] < props['TOTAL_POPULATION']
```
But this would require verification from FEMA on what these fields mean.

Reply:
I’ll reach out to our FEMA contact